src/components/bookCard/BookCard.tsx

```diff
@@ -0,0 +1,12 @@
+import './BookCard.css'
+
+function BookCard() {
+  return (
+    <>
```

Comment:
This fragment is not needed since return statement would still have one top level element without it.